### DISCUSSION TITLE
Native adaptive mesh klipper support

### DISCUSSION THREAD

**Discussion by KaratsuFr** [jschuh/klipper-macros#238] [Ideas]
klipper recently add adaptive mesh 
can you update your wonderfall script to make use of native adaptive mesh ?
https://www.klipper3d.org/Bed_Mesh.html#adaptive-meshes

**Comment by jschuh** [comment]
Yeah, in principle it's a small change because the logic between the native implementation and the macros is pretty much identical (accepting that I never got around to supporting deltas). However, there are some things that would require extra testing (e.g. random placement) and I just have much time right now to dig into it.

**Reply by KaratsuFr** [reply]
if i can be of any help, i don t have a delta but an ender3 + sonic pad  if that help
